public/javascripts/SVLabel/src/SVLabel/onboarding/OnboardingStates.js

```diff
@@ -590,7 +642,7 @@ function OnboardingStates (compass, mapService, statusModel, tracker) {
             "transition": function () {
                 tracker.push('Onboarding_Transition', {onboardingTransition: "redo-tag-attribute-2"});
```

Comment:
Just another tracker change

```suggestion
                tracker.push('Onboarding_Transition', {onboardingTransition: "redo-tag-attribute-3"});
```